src/renderer/components/dock/dock.tsx

```diff
@@ -105,7 +105,7 @@ export class Dock extends React.Component<Props> {
     if (!isOpen || !selectedTab) return null;
 
     return (
-      <div className="tab-content" style={{ flexBasis: height }}>
+      <div className="tab-content" style={{ flexBasis: isOpen ? height : 0 }}>
```

Comment:
what is the point of this? if `isOpen` is false then the above `if` will `return null`

Reply:
Well, previously it was done for animation when closing Dock.. Here doesn't work as intended, true.

Reply:
Removed.